.changeset/late-toes-cry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+fixed the generated namespace for "embedded" sveltekit apps so that two separate pages can be inlined into the same page without clobbering each other
```

Comment:
fix messages in the changelog should start with `fix: ` for consistency: https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md

sveltekit should be capitalized as SvelteKit